cl/users/models.py

```diff
@@ -224,13 +224,11 @@ class EmailFlag(AbstractDateTimeModel):
     SMALL_ONLY = 0
     MAX_RETRY_REACHED = 1
     FLAGS = (
-        (SMALL_ONLY, "small_email_only"),
-        (MAX_RETRY_REACHED, "max_retry_reached"),
+        (SMALL_ONLY, "Small Email Only"),
+        (MAX_RETRY_REACHED, "Max Retry Reached"),
```

Comment:
I know I made this suggestion but was `small_email_only` a code from AWS like the above?

Reply:
I keep these changes since these values are not sent by the SES notification, we assign them locally, so I think is more understandable like this.